recommendations/forms.py

```diff
@@ -3,7 +3,29 @@
 
 
 class GenreForm(forms.Form):
+    Question = forms.CharField(
+        initial="Let's pick your desired genre(s)!", disabled=True, max_length=50
+    )
+
     OPTIONS = GENRES
     Genres = forms.MultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple,
+        choices=OPTIONS,
+        help_text=(
```

Comment:
I think it could be nice to also have the option (via a checked box or something) of searching for movies that "contains at least 1 of these genres"

Reply:
agreed! a pipe dream that i should definitely pursue :)
for now, my want is to try and narrow the possible movies down as much as possible